my-app/src/App.css

```diff
@@ -4,21 +4,31 @@
 @import url('https://fonts.googleapis.com/css2?family=Eczar:wght@400;500;600;700;800&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=IM+Fell+English+SC&family=IM+Fell+English:ital@0;1&display=swap');
 
+/* DEFAULT STYLING (MOBILE) */
+
 body {
   background-color: #fefefe;
   background-image: url("https://www.transparenttextures.com/patterns/exclusive-paper.png");
```

Comment:
I think it's worth downloading this texture so we don't rely on an external URL.

Reply:
Great shout. Done.